scripts/utils/changePackages.ts

```diff
@@ -21,7 +21,9 @@ export async function changeAllPkgJSON(version: string) {
   const spinner = ora('Bumping versions').start();
 
   for (const dir of dirs) {
-    changePkgJsonVersion(`./packages/${dir}`, version);
+    if (dir.includes('fuels-ts')) {
```

Comment:
Why if requires this fuels-ts?

Reply:
this is to avoid updating the `forc-bin` package.json file